app/components/description.hbs

```diff
@@ -1,40 +1,18 @@
 {{#if @isDev}}
-  <section class="description--dev" data-test-description-section>
-    <h2 class="description__title--dev" data-test-description-section-title>What
-      is Real Dev Squad?</h2>
-    <div
-      class="description__content--dev"
-      data-test-description-section-content
-    >
-      <p data-test-description-section-content="para-first">
-        Real Dev Squad is an online non-profit open source free fun community
-        for people in tech, mainly developers, designers, college students, or
-        product managers, to come, learn and contribute towards building a
-        platform for our community, that helps upskill everyone.
-      </p>
-      <p data-test-description-section-content="para-second">
-        We are an inclusive, respectful, warm, motivated and commited squad of
-        people who constantly grow together and tackle bigger and harder
-        challenges to ensure we become some of the best problem solvers,
-        engineers, designers and more out there.
-      </p>
-    </div>
-  </section>
-{{else}}
-  <section class="description">
+<section class="description--dev">
```

Comment:
Here if you are removing ff then what is needed for this to be in ff?

Reply:
added old one in `dev=true` will remove after release